lib/game.rb

```diff
@@ -0,0 +1,50 @@
+class Game
+  include Concerns::Winable
+  attr_accessor :board, :player_1, :player_2
+
+  def initialize(player_1 = Human.new("X"), player_2 = Human.new("O"), board = Board.new)
+    @board = board
+    @player_1 = player_1
+    @player_2 = player_2
+  end
+
+  def current_player
+    board.turn_count % 2 == 0 ? player_1 : player_2
+  end
+
+  def over?
+    draw? || won?
+  end
+
+  def draw?
+    board.full? && !won?
+  end
+
+  def winner
+    board.cells[won?.first] if won?

```

Comment:
I think I wanted the expression in one line at the time. I need to learn when to avoid one liners.